### PR TITLE
add presets page

### PR DESCRIPTION
**Pull request by hzoo**

### CODE REVIEW

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit 4648c716fd40f61f04436364f52ca8f6d6a86e46

https://deploy-preview-1720--babel-new.netlify.com

File: docs/cli.md [hunk 1]
```diff
@@ -4,21 +4,149 @@ title: babel-cli
...
+```
+
+## Basic Usage 
 
 ```sh
 babel script.js
 ```
 
-For more in depth documentation see: http://babeljs.io/docs/usage/cli/
+## babel
```
Comment (microbouji) [inline docs/cli.md]: With babel-node being a separate page this heading is probably not needed anymore.

File: docs/cli.md [hunk 2]
```diff
@@ -4,21 +4,149 @@ title: babel-cli
...
+
+Ignore the configuration from the projects .babelrc file and use the cli options e.g. for a custom build
+
+```sh
+npx babel --no-babelrc script.js --out-file script-compiled.js --presets=es2015,react
+```
+
+### Advanced Usage
 
+There are many more options available in the babel CLI, see [options](/docs/usage/api/#options), `babel --help` and other sections for more information.
```
Comment (microbouji) [inline docs/cli.md]: new options link `core.md#options`

File: docs/plugins.md [hunk 3]
```diff
@@ -191,84 +144,66 @@ These plugins are in the minify repo.
...
-shorthand
-
-`"presets": ["@org/name"]`
+```js
+{
+  "plugins": [
+    "@org/babel-plugin-name",
+    "@org/name" // equivalent
+  ]
+}
```
Comment (microbouji) [inline docs/plugins.md]: code block needs closing fence here

File: docs/presets.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+This also works with scoped packages:
+
+```js
+{
+  "presets": [
+  	"@org/babel-preset-name",
+  	"@org/name" // equivalent
+  ]
+}
```
Comment (microbouji) [inline docs/presets.md]: code block needs closing fence here

File: docs/presets.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
+---
+id: presets
+title: Presets
+---
+
+Don't want to assemble your own set of plugins? No problem! Presets can act as an array of Babel plugins or even a sharable [`.babelrc`](babelrc.md) config.
+
+### Official Presets
```
Comment (microbouji) [inline docs/presets.md]: if this is made into an h2 instead of h3 it will show up in the page nav too

File: docs/presets.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+We've assembled some for common environments:
+
+- [@babel/preset-env](preset-env.md)
+- [@babel/preset-flow](preset-flow.md)
+- [@babel/preset-react](preset-react.md)
+- [@babel/preset-typescript](preset-typescript.md)
+
+> Many other community maintained presets are available [on npm](https://www.npmjs.com/search?q=babel-preset)!
+
+### Stage-X (Experimental Presets)
```
Comment (microbouji) [inline docs/presets.md]: if this is made into an h2 instead of h3 it will show up in the page nav too

**Comment by hzoo** [comment]
@microbouji feel free to make those changes if you'd like (might not get to it)!

**Comment by microbouji** [comment]
done!

**Pull request merged**